pkgs/development/python-modules/deezer-py/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+, eventlet
+}:
+
+buildPythonPackage rec {
+  pname = "deezer-py";
+  version = "0.0.14";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "+svakCSDGxUYC6wfMWXAlLdNwdbTsSrxS/v11OGwgwY=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    eventlet
+  ];
+
+  doCheck = false; # tcp: protocol not found
+
+  meta = with lib; {
+    description = "A wrapper for all Deezer's APIs";
+    homepage = "https://git.rip/RemixDev/deezer-py";
+    license = licenses.gpl3Plus;
+    maintainers = with maintainers; [ luc65r ];
+    platforms = platforms.all;
```

Comment:
```suggestion
```

set by buildPythonPackage

Reply:
Even though this may be the case, I feel like `platforms` should be set explicitly in all packages so that, when using plain `stdenv.mkDerivation` (or any wrapper that doesn't automagically set `platforms`), people don't forget to set it. The [manual states that "Platforms should be set"](https://nixos.org/manual/nixpkgs/stable/#reviewing-contributions-new-packages) (admittedly for the purpose of getting cached by Hydra).

If somebody who is new to Nixpkgs bases their package on something that is `buildPythonPackage` or `buildGoModule` (but doesn't use a wrapper like those), they might not see `platforms` getting set and do the same, and (if uncaught) end up wondering why their package isn't being substituted.

Reply:
It can't be set to platforms.all here because go, python or rust do not support platforms.all which would create a lot of packages with dependencies which are not available. Please remove it.

Also if people just take any random package which is not using the same wrapper as their package they are doing something wrong. You should choose a package as a template which is using the same language and wrapper as your package.